README.md

```diff
@@ -1,9 +1,13 @@
-# Claudie
-
-[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![Go Report Card](https://goreportcard.com/badge/github.com/Berops/claudie)](https://goreportcard.com/report/github.com/Berops/claudie)
-
-Platform for managing multi-cloud Kubernetes clusters with each nodepool in a different cloud-provider.
+<h4 align="center">
+  <img src="https://raw.githubusercontent.com/berops/claudie/4160faff3c2430aec2e6de4b1a6698c6b7df7f2d/docs/logo%20claudie%20blue.svg" width="400px"/><br/>
```

Comment:
this link probably won't last, I need to find some other way

Reply:
I think I'll merge this and just fix this if it doesn't work, so ignore it please